api/internal/repository/story_content.go

```diff
@@ -0,0 +1,75 @@
+package repository
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+)
+
+type StoryContentRepository interface {
+	GetContentByChapter(franchiseId int64, storyNum int, chapterNum int) ([]*StorySectionModel, error)
+}
+
+type StorySectionModel struct {
+	Type string             `json:"type"`
+	Data []*StoryMediaModel `json:"data"`
+}
+
+type StoryMediaModel struct {
+	Type        string `json:"type"`
+	Content     string `json:"content"`
+	Url         string `json:"url"`
+	Description string `json:"description,omitempty"`
+}
+
+func NewStoryContentFsImpl(contentFilePath string) (StoryContentRepository, error) {
```

Comment:
Just for curiosity, does it mean we would plan to support both file and DB to store/load story content?

Reply:
Only file at this moment since the data is small. We can directly load into the memory and serve fast, even faster than using cache like redis since we don't need to make network calls.